packages/components/package.json

```diff
@@ -34,14 +34,14 @@
   "devDependencies": {
     "@ant-design/icons": "4.7.0",
     "@babel/core": "7.17.8",
-    "@monokle/validation": "*",
+    "@monokle/validation": "0.12.0",
```

Comment:
There should be no need to do this

Reply:
You mean the exact version number? I was getting import errors in storybook with version 13 and 14, but maybe I was just doing something wrong.

Reply:
https://github.com/kubeshop/monokle-core/pull/285#issuecomment-1463531076

Reply:
@monojack `node-fetch` is a dependency of storybook so shouldn't it be installed automatically without us having to add it to our dependencies?

Reply:
@DzoQiEuoi, yes, but the same `3.3.0` version is also a dependency of the `validation` package and turbo moves `node-fetch` to the root of the monorepo. To make things worse, storybook also has some dependencies deep down it's dependency tree that need `node-fetch@2.x` and I think this is where it gets a bit funky, because `v2.x` versions seem to be deduped for some reason and don't end up being installed at all. Installing node-fetch locally (regardless of the version) will influence (cba how) the deduping process and the `v2.x` versions get installed where they should, fixing the problem. Same with `uuid`

Reply:
P.S. this only happens predominantly in monorepos, if you were to extract the whole `components` package out, I'm sure this wouldn't be an issue. monorepos ftl